### PR TITLE
Adding playbook for ODCS deployment

### PR DESCRIPTION
**Pull request by mohanboddu**
This is a WIP, please hold on merging until https://github.com/CentOS/ansible-role-odcs-backend/pull/1 and https://github.com/CentOS/ansible-role-odcs-frontend/pull/1 are merged.

Signed-off-by: Mohan Boddu <mboddu@bhujji.com>

### CODE REVIEW

**Comment by arrfab** [comment]
LGTM so we can already merge this, as it's independent from the work being done on these two roles, in their own git repositories.
So holding for now but imho this could be merged already :)

File: role-odcs.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+- name: Setup ODCS frontend
```
Comment (arrfab) [inline role-odcs.yml]: can we have role-odcs-frontend.yml and role-odcs-backend.yml, themselves calling corresponding role ? Just to keep the logic we use so far, as automated ansible deploy will try to ensure each git repo/role is up2date before kicking ansible-playbook for the role

And BTW, that's how it's declared in role-all.yml : two different playbooks, so they should exist and the role-odcs.yml deleted

Comment (mohanboddu) [reply role-odcs.yml]: Thanks @arrfab, I fixed it

**Comment by arrfab** [comment]
@mohanboddu : thanks a lot for the change. Happy to merge it if that also works for you (as we don't need fully tested role for this PR)

**Review by mohanboddu — COMMENTED** [review]
> @mohanboddu : thanks a lot for the change. Happy to merge it if that also works for you (as we don't need fully tested role for this PR)

Sure, go ahead

**Pull request merged**